app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -74,6 +86,46 @@ class ItemListFragment : CommonFragment(), ItemListView {
                 .addTo(compositeDisposable)
     }
 
+    private fun setupItemListSortMenu(sortButton: Button) {
+        context?.let {
+            val theme = ContextThemeWrapper(it, R.style.PopupMenu)
+            popupMenu = PopupMenu(theme, sortButton)
+            val menuInflater: MenuInflater = popupMenu.menuInflater
+            menuInflater.inflate(R.menu.sort_menu, popupMenu.menu)
```

Comment:
Constructing this this way looks a little weird.

Can we get the inflater directly from the activity? Do you have to construct the `PopupMenu` in code like this?